bootstrap/lib/notificationProxy.js

```diff
@@ -28,7 +28,7 @@ exports.adminNotificationWebsocketRouter = function(context) {
       }
 
       router.use(function abc(req,res,next) {
-        context.logger.info('Saw Websocket request, method='+req.method);
+        context.logger.info('ZWED5001I - Saw Websocket request, method='+req.method);
```

Comment:
This will not work, because you do not have a translation JSON for any lib or nodeserver content.